src/main/java/com/openlattice/chronicle/controllers/ChronicleStudyController.java

```diff
@@ -130,15 +132,50 @@ public Boolean isKnownParticipant(
         return chronicleService.isKnownParticipant( studyId, participantId );
     }
 
-    @Override
-    public void deleteParticipantAndAllNeighbors(
-            UUID studyId, String participantId, DeleteType deleteType ) {
-        // TODO implement this
+
+    @RequestMapping(
```

Comment:
Are these delete calls only used for testing? I'm a little nervous about adding them to a production/staging controller if they're only for testing (tho I'm not that familiar with Chronicle)

Reply:
Oh derp, I see now that this is hard delete 👍